cryptography/src/commonJvmAndroid/kotlin/com.wire.kalium.cryptography/MLSClientImpl.kt

```diff
@@ -326,14 +326,14 @@ class MLSClientImpl(
             return clientId?.let {
                 WireIdentity(
                     CryptoQualifiedClientId.fromEncodedString(value.clientId)!!,
-                    value.handle,
-                    value.displayName,
-                    value.domain,
-                    value.certificate,
+                    value.x509Identity?.handle ?: "",
+                    value.x509Identity?.displayName ?: "",
+                    value.x509Identity?.domain ?: "",
+                    value.x509Identity?.certificate ?: "",
```

Comment:
please do not add default empty strings, if the value is supposed to be nullable now then keep is nullable